apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeCard.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Box,
+  Button,
+  Card,
+  FormControl,
+  Select,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { css } from 'emotion';
+import { AllContentTypes, ContentTypeEntries } from 'types';
+
+interface Props {
+  allContentTypes: AllContentTypes;
+  contentTypeEntries: ContentTypeEntries;
+  onContentTypeChange: (prevKey: string, newKey: string) => void;
+  onContentTypeFieldChange: (key: string, field: string, value: string) => void;
+  onAddContentType: () => void;
+  onRemoveContentType: (key: string) => void;
+}
+
+const styles = {
```

Comment:
Thinking about how this could be done without having a hidden `remove` header cell and came up with this:
```
const styles = {
  contentTypeItem: css({
    flex: 4,
  }),
  removeItem: css({
    flex: 1,
  }),
  hidden: css({
    visibility: 'hidden',
  }),
};
```

Where the input field cells and header cells are `contentTypeItem`, and the last (now empty)`Box` in the header and all the remove Box cells are `removeItem`.


...I'm not convinced my approach is any better, I'll let you decide what you like more. 😅

Reply:
Thank you for this! I also didn't love having the hidden header cell, so I will update to your approach.